airbyte-integrations/connectors/source-the-guardian-api/unit_tests/pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.poetry]
+name = "source-the-guardian-api-tests"
+version = "0.0.0"
+description = "Unit tests for source-the-guardian-api"
+authors = ["Airbyte <contact@airbyte.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+airbyte-cdk = "5.17.0"
+pytest = "^8"
```

Comment:
Two things to note:

1. I opted out of dev dependencies as the whole project is just a testing harness and I don't believe there is any need for separation of main/dev deps.
2. For today, the version will have to be manually matched to whatever is listed in the connector's `metadata`. Ideally the next step will be to add some logic to up-to-date that automatically does this for us.